07 matrices/Matrix_Division_Multiplication.java

```diff
@@ -96,9 +98,9 @@ void multi() {
 	public static void main(String args[]) {
 		
 		Matrix_Division_Multiplication obj = new Matrix_Division_Multiplication();
-		
 		obj.create();
 		obj.display();
 		obj.div();
+		obj.multi();
```

Comment:
This approach looks really good. I have a question, what do you think about obj reference?

Reply:
Well, small examples I think it isn't to bad, but when you're in biggest projects I think it's a smell code, because have a lot of code line to read and it turn to difficult for understand.

Reply:
Ok thank you!